src/js/map.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable object-shorthand */
+/* eslint-disable prefer-template */
+/* eslint-disable func-names */
+/* eslint-disable no-undef */
+mapboxgl.accessToken = 'pk.eyJ1IjoidmVyYXNpYXZrYSIsImEiOiJja3JkaXk1N3k1Y2k5Mm9xcDNiNnJhZ3czIn0.VVSKa_qR-1yqHrKVi7XE0w';
+
+const latitude = JSON.parse(window.localStorage.getItem('latitude'));
+const longitude = JSON.parse(window.localStorage.getItem('longitude'));
+const map = new mapboxgl.Map({
+  container: 'map',
+  style: 'mapbox://styles/mapbox/streets-v11',
+  center: [`${longitude}`, `${latitude}`],
+  zoom: 8,
+});
+
+/* Given a query in the form "lng, lat" or "lat, lng"
+* returns the matching geographic coordinate(s)
+* as search results in carmen geojson format,
+* https://github.com/mapbox/carmen/blob/master/carmen-geojson.md */
+const coordinatesGeocoder = function (query) {
+// Match anything which looks like
+// decimal degrees coordinate pair.
+  const matches = query.match(/^[ ]*(?:Lat: )?(-?\d+\.?\d*)[, ]+(?:Lng: )?(-?\d+\.?\d*)[ ]*$/i);
+  if (!matches) {
+    return null;
+  }
+
+  function coordinateFeature(lng, lat) {
+    return {
+      center: [lng, lat],
+      geometry: {
+        type: 'Point',
+        coordinates: [lng, lat],
+      },
+      place_name: 'Lat: ' + lat + ' Lng: ' + lng,
+      place_type: ['coordinate'],
+      properties: {},
+      type: 'Feature',
+    };
+  }
+
+  const coord1 = Number(matches[1]);
+  const coord2 = Number(matches[2]);
+  const geocodes = [];
+
+  if (coord1 < -90 || coord1 > 90) {
+    // must be lng, lat
+    geocodes.push(coordinateFeature(coord1, coord2));
+  }
+
+  if (coord2 < -90 || coord2 > 90) {
+    // must be lat, lng
+    geocodes.push(coordinateFeature(coord2, coord1));
+  }
+
+  if (geocodes.length === 0) {
+    // else could be either lng, lat or lat, lng
+    geocodes.push(coordinateFeature(coord1, coord2));
+    geocodes.push(coordinateFeature(coord2, coord1));
+  }
+
+  return geocodes;
+};
+
```

Comment:
Not sure what we are doing there. No errors - but we can discuss it.

Reply:
this is code from mapbox api)